components/window/actions.md

```diff
@@ -65,6 +65,7 @@ Action buttons expose the following properties:
 }
 ````
 
+>Setting a custom icon for a built-in action is not supported - the Window will override it and use the default SVG icon for the corresponding built-in action. If you need to specify a custom icon, use a [custom action](#custom-actions) instead of the built-in one. 
```

Comment:
```suggestion
>Setting a custom icon for a built-in action is not supported - the Window will override it and use the default icon for the corresponding built-in action. If you need to specify a custom icon, use a [custom action](#custom-actions) instead of the built-in one. 
```

Although SVG is the default icon type, the Window will still render built-in font icons as long as this is how the app is configured, hence it might be best to omit the icon type.